install.sh

```diff
@@ -154,7 +125,25 @@ case "${ARCH}" in
   ;;
 esac
 
-# create the device.json file if it doesn't exist
+# Get the URLs and hashes of the bootstrap packages
+BOOTSTRAP_PACKAGES="ruby git ca_certificates"
+for package in $BOOTSTRAP_PACKAGES; do
+  curl -O "${URL}/packages/${package}.rb"
+  [[ "$(sed -n '/binary_sha256/,/}/p' "${package}.rb")" =~ .*${ARCH}:[[:blank:]]*[\'\"]([^\'\"]*) ]]
+    sha256s+=("${BASH_REMATCH[1]}")
+
+  name=$(sed -n '3s/class\(.*\)<.*/\L\1/;3s/^[ \t]*//;3s/[ \t]*$//p' "${package}.rb")
+  version=$(sed -n '6s/version \(.*\)/\1/;6s/^[ \t]*//;6s/[ \t]*$//p' "${package}.rb" | tr -d "'")
```

Comment:
What happens when one of the bootstrap packages is bastardized with `@_ver` and no longer represents the true version?

Reply:
Bad things. 
That's why I've tried to limit the amount of bootstrap packages so not that many need to stick to not using `@_ver`

Reply:
I'm still not sold on this kludgy solution.

Reply:
What if you had logic that searches for a `@_ver` line and uses that version, otherwise use the `version` line value?  This assumes that every package is using `@_ver` for that purpose.

Reply:
I could probably add logic that searches for an `@_ver` line, takes the value of that, then suffixes the value of the `version "#{@_ver}-x"` onto it, or we could just not use `@_ver` for 5 packages/at all.